safe/definitions/post_processors/__init__.py

```diff
@@ -43,5 +43,6 @@
      age_vulnerability_postprocessors +
      disabled_vulnerability_postprocessors +
      gender_vulnerability_postprocessors) + [
-    post_processor_additional_rice
+    post_processor_additional_rice,
+    post_processor_productivity
```

Comment:
__[flake8]__
* [F405] 'post_processor_productivity' may be undefined, or defined from star imports: minimum_needs_post_processors, population_post_processors, post_processor_functions, post_processor_inputs, post_processors

<!-- Comment by [SideCI](https://sideci.com) -->